### PR TITLE
 Fix Bugs in pagerank & idd implementation

### PR DESCRIPTION
**Pull request by Sakeerthan**

### CODE REVIEW

File: src/server/JasmineGraphInstanceService.cpp [hunk 1]
```diff
@@ -1689,30 +1691,49 @@ map<long, double> calculateLocalPageRank(string graphID, double alpha, string pa
...
+    std::string partitionCount = Utils::getJasmineGraphProperty("org.jasminegraph.server.npartitions");
+    int parCount = std::stoi(partitionCount);
+
+    for (int partitionID = 0; partitionID < parCount; ++partitionID) {
+        std::string iddFilePath = aggregatorFilePath + "/" + graphID + "_idd_" + std::to_string(partitionID);
+        std::ifstream dataFile;
+        dataFile.open(iddFilePath);
+
+        while (!dataFile.eof()) {
+            std::string str;
```
Comment (miyurud) [inline src/server/JasmineGraphInstanceService.cpp:1707]:
```suggestion
            std::string line;
```

Comment (miyurud) [reply src/server/JasmineGraphInstanceService.cpp:1707]: Refactor the remaining occurrences of `str` with `line`.

**Review by miyurud — CHANGES_REQUESTED** [review]
There is some changes required.

File: src/server/JasmineGraphInstanceService.cpp (hunk 1 above)
Comment (Sakeerthan) [reply src/server/JasmineGraphInstanceService.cpp]: Done.

Comment (miyurud) [reply src/server/JasmineGraphInstanceService.cpp]: Ok

**Review by miyurud — APPROVED** [review]

**Pull request merged**